tests/functional/test-no-offer.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+project_id=75681640118a4890b3da0d106eae8af7 # test1 uuid
+tmpfile=$(mktemp ./tmpXXXXXX)
+start=$(date +%Y-%m-%d)
+end=$(date -d "+5 days" +%Y-%m-%d)
+nodefile=$(mktemp /tmp/nodes/XXXX)
+errfile=$(mktemp ./errXXXXXX)
+node_uuid=$(echo $nodefile | sed 's/\/tmp\/nodes\///')
+
+trap "rm -f $nodefile $tmpfile $errfile" EXIT
+
+# Create dummy node
+
+cat <<EOF > /tmp/nodes/$node_uuid
+{
+    "project_owner_id": "$project_id",
+    "server_config": {
+        "new attribute XYZ": "This is just a sample list of free-form attributes used for describing a server.",
+        "cpu_type": "Intel Xeon",
+        "cores": 16,
+        "ram_gb": 512,
+        "storage_type": "samsung SSD",
+        "storage_size_gb": 204
+    }
+}
+EOF
```

Comment:
Is it possible to genericize setup.sh somehow to cover this node/temp file creation?

Reply:
setup.sh creates an offer as part of the script, so I didn't want to use it for the no-offer script
I didn't expect I'd need any other scripts that don't create an offer when they're being set up, so I didn't think I should move this to its own script

Reply:
okay! I think this could probably be restructured somewhat, but that can be done later - it's fine for now!